code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -411,9 +412,18 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 
 /datum/gas_mixture/react(turf/open/dump_location)
 	. = NO_REACTION
+	if(volume == 0)

```

Comment:
This is good

Reply:
Just if(!volume)

Reply:
I'd rather see `if(volume <= 0)`, because if everything is well optimized in the CPU it will just check for the first bit and save some operation time.